scalapact-scalatest/src/main/scala/com/itv/scalapact/model/ScalaPactOptions.scala

```diff
@@ -2,9 +2,14 @@ package com.itv.scalapact.model
 
 import scala.util.Properties
 
-final case class ScalaPactOptions(writePactFiles: Boolean, outputPath: String)
+final case class ScalaPactOptions(writePactFiles: Boolean, outputPath: String, host: String, port: Int)
```

Comment:
could you add an auxiliary apply method on the companion object to at least maintain source compatibility?
```
def apply(writePactFiles: Boolean, outputPath: String) = 
  ScalaPactOptions(writePactFiles, outputPath, "localhost", 0)
```

Reply:
Done :)